packages/node-cli/src/commands/ManageAccountCommandGenerator.js

```diff
@@ -0,0 +1,263 @@
+   REVOKE: "revoke",
+};
+
+const COMMAND = {
+   OPTIONS: {
+      INFO: "info",
+      LIST: "list",
+      REMOVE: "remove",
+      RENAME: "rename",
+      RENAMETO: "renameto",
+   },
+};
+
+const ANSWERS_NAMES = {
+   SELECTED_AUTH_ID: "selected_auth_id",
+   ACTION: "action",
+   AUTHID: "authId",
+   RENAMETO: "renameto",
+   REMOVE: "remove",
+};
+
+const {
+   showValidationResults,
+   validateAuthIDNotInList,
+   validateAlphanumericHyphenUnderscore,
+   validateFieldHasNoSpaces,
+   validateFieldIsNotEmpty,
+   validateMaximumLength,
+   validateSameAuthID,
+} = require("../validation/InteractiveAnswersValidator");
+
+const PROPERTIES = {
+   INFO: "info",
+   ACCOUNT_INFO: "accountInfo",
+   URLS: "urls",
+};
+
+const DOMAIN = "domain";
+module.exports = class ManageAccountCommandGenerator extends BaseCommandGenerator {
+   constructor(options) {
+      super(options);
+      this._authenticationService = new AuthenticationService(options.executionPath);
+      this._outputFormatter = new ManageAccountOutputFormatter(options.consoleLogger);
+   }
+
+   async _getCommandQuestions(prompt) {
+      const authIDList = await this._authenticationService.getAuthIds(this._sdkExecutor);
+      let answers = await this._selectAuthID(authIDList, prompt);
+      this._outputFormatter.logAccountCredentials(answers[ANSWERS_NAMES.SELECTED_AUTH_ID]);
+      const selectedAuthID = answers[ANSWERS_NAMES.SELECTED_AUTH_ID].authId;
+      answers[ANSWERS_NAMES.ACTION] = await this._selectAction(prompt);
+      if (answers[ANSWERS_NAMES.ACTION] == ACTION.RENAME) {
+         answers[ANSWERS_NAMES.RENAMETO] = await this._introduceNewName(prompt, authIDList, selectedAuthID);
+      } else if (answers[ANSWERS_NAMES.ACTION] == ACTION.REMOVE) {
+         answers[ANSWERS_NAMES.REMOVE] = await this._confirmRemove(prompt);
+      }
+
+      return this._extractAnswers(answers);
+   }
+
+   async _selectAuthID(authIDList, prompt) {
+      var authIDs = Object.entries(authIDList).sort();
+      if (authIDs.length <= 0) {
+         throw NodeTranslationService.getMessage(ERRORS.CREDENTIALS_EMPTY);
+      }
+      const choices = [];
+      authIDs.forEach((authIDArray) => {
+         const authID = authIDArray[0];
+         const accountCredential = authIDArray[1];
+         const accountCredentialString = this.outputFormatter.accountCredentialToString(authID, accountCredential);
+         choices.push({
+            name: accountCredentialString,
+            value: { authId: authID, accountInfo: accountCredential.accountInfo, domain: accountCredential.urls.app },
+         });
+      });
+      choices.push(new inquirer.Separator());
+      let answers = await prompt([
+         {
+            type: CommandUtils.INQUIRER_TYPES.LIST,
+            name: ANSWERS_NAMES.SELECTED_AUTH_ID,
+            message: NodeTranslationService.getMessage(MESSAGES.SELECT_CONFIGURED_AUTHID),
+            choices: choices,
+         },
+      ]);
+      return answers;
```

Comment:
I think we can directly `return prompt([...]);` . We are already use `await` to get the result of this method.